example/android/app/src/androidTest/java/com/example/InstabugSample/util/InstabugAssertions.java

```diff
@@ -0,0 +1,19 @@
+package com.example.InstabugSample.util;
+
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static org.junit.Assert.assertTrue;
+
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiSelector;
+
+public class InstabugAssertions {
```

Comment:
**suggestion:** The prefix `Instabug` does not serve any purpose here, the methods in this class seem to be abstract enough. Consider using a non-prefixed name like `AssertionUtils`.

Reply:
It was named that way to be consistent with `InstabugViewMatchers`, kinda like `ViewMatchers` and `ViewAssertions` from Espresso.
Even though it doesn't have Instabug-specific assertions at the moment, it might have in the future _(if we end up using native UI tests)_.